options.js

```diff
@@ -77,6 +78,10 @@ function saveOptions() {
             options.lateScoreAdjustment = 0;
             needsReload = true;
         }
+        if (options.students.trim() !== options.students) {
+            options.students = options.students.trim();
+            needsReload = true;
+        }
```

Comment:
If `needsReload = true` is unnecessary, could just use `options.students = options.students.trim()`